arch/x86/kvm/vmx/pkvm/include/pkvm.h

```diff
@@ -111,7 +113,8 @@ struct pkvm_section {
 #define PKVM_PAGES (ALIGN(sizeof(struct pkvm_hyp), PAGE_SIZE) >> PAGE_SHIFT)
 #define PKVM_PCPU_PAGES (ALIGN(sizeof(struct pkvm_pcpu), PAGE_SIZE) >> PAGE_SHIFT)
 #define PKVM_HOST_VCPU_PAGES (ALIGN(sizeof(struct pkvm_host_vcpu), PAGE_SIZE) >> PAGE_SHIFT)
-#define PKVM_VMCS_PAGES 3 /*vmxarea+vmcs+msr_bitmap*/
+#define PKVM_HOST_VCPU_VMCS_PAGES 3 /*vmxarea+vmcs+msr_bitmap*/
+#define PKVM_EXTRA_PAGES 2 /*host vm io bitmap*/
```

Comment:
can it combine with PKVM_PAGES?

Reply:
PKVM_PAGES is used to allocate pages for pkvm_hyp structure. Beacuse the io bitmap and the later mmcfg list table is not added to pkvm_hyp, don't think we can use it?

Reply:
Okay, got it.